tsdb/engine/tsm1/timestamp_test.go

```diff
@@ -28,7 +28,7 @@ func Test_TimeEncoder(t *testing.T) {
 		t.Fatalf("Wrong encoding used: expected rle, got %v", got)
 	}
 
-	dec := NewTimeDecoder()
+	var dec TimeDecoder
```

Comment:
Go will use the pointer if the function's receiver is a pointer. So it automatically converts:

```go
dec.Init(b)
```

to

```go
(&dec).Init(b)
```